cybsi/api/threat_landscape/api.py

```diff
@@ -0,0 +1,638 @@
+                Provided value are invalid (see args value requirements).
+            :class:`~cybsi.api.error.NotFoundError`:
+                Threat landscape or custom list not found.
+            :class:`~cybsi.api.error.SemanticError`:
+        Note:
+            Semantic error codes specific for this method:
+              * :attr:`~cybsi.api.error.SemanticErrorCodes.DictionaryNotFound`
+              * :attr:`~cybsi.api.error.SemanticErrorCodes.DictionaryMismatch`
+
+        """
+        params: Dict[str, Any] = {"dictionaryUUID": str(dictionary_uuid)}
+        path = (
+            f"{_PATH}/{landscape_uuid}/custom-lists/"
+            f"{custom_list_uuid}/related-dictionaries"
+        )
+        self._connector.do_post(path=path, json=params)
+
+    def delete_related_dictionary(
+        self,
+        landscape_uuid: uuid.UUID,
+        dictionary_uuid: uuid.UUID,
+        custom_list_uuid: uuid.UUID,
+    ) -> None:
+        """
+        Add related dictionary to custom list of Threat landscape.
+
+        .. versionadded:: 2.14.0
+        Note:
+            Calls `DELETE /threat-landscapes/{landscapeUUID}/custom-lists/
+                    {customListUUID}/related-dictionaries/{dictionary_uuid}/related-dictionaries/{dictionary_uuid}`
+        Args:
+            landscape_uuid: Landscape UUID
+            dictionary_uuid: Dictionary UUID
+            custom_list_uuid: Custom list UUID
+        Raises:
+            :class:`~cybsi.api.error.InvalidRequestError`:
+                Provided value are invalid (see args value requirements).
+            :class:`~cybsi.api.error.NotFoundError`:
+                Threat landscape or custom list not found.
+
+        """
+        path = (
+            f"{_PATH}/{landscape_uuid}/custom-lists/"
+            f"{custom_list_uuid}/related-dictionaries/{dictionary_uuid}"
+        )
+        self._connector.do_delete(path=path)
+
+    def build_query(
+        self,
+        landscape_uuid: uuid.UUID,
+        *,
+        data_source_uuid: Optional[uuid.UUID] = None,
+    ) -> str:
+        """
+        Build a query for a Threat landscape.
+
+        .. versionadded:: 2.14.0
+        Note:
+            Calls `GET /threat-landscapes/{landscapeUUID}/query`
+        Args:
+            landscape_uuid: Landscape UUID
+            data_source_uuid: Data source UUID
+        Raises:
+            :class:`~cybsi.api.error.InvalidRequestError`:
+                Provided value are invalid (see args value requirements).
+            :class:`~cybsi.api.error.NotFoundError`: Threat landscape not found.
+            :class:`~cybsi.api.error.SemanticError`:
+        Note:
+            Semantic error codes specific for this method:
+            * :attr:`~cybsi.api.error.SemanticErrorCodes.DataSourceNotFound`
+            * :attr:`~cybsi.api.error.SemanticErrorCodes.EmptyLandscape`
+        """
+        params: Dict[str, Any] = {}
+        if data_source_uuid:
+            params["dataSourceUUID"] = data_source_uuid
```

Comment:
Давай здесь обернём в str
```suggestion
            params["dataSourceUUID"] = str(data_source_uuid)
```